x/nft/ticker/handler.go

```diff
@@ -28,10 +28,10 @@ type IssueHandler struct {
 	auth        x.Authenticator
 	issuer      weave.Address
 	bucket      Bucket
-	blockchains blockchain.Bucket
+	blockchains orm.Bucket
```

Comment:
pp: I assume you replace the type to prevent dependency issues but an interface would be a nicer abstraction.